MapboxNavigation/NavigationViewController.swift

```diff
@@ -537,19 +537,6 @@ extension NavigationViewController: RouteControllerDelegate {
         }
     }
     
-    @objc public func routeController(_ routeController: RouteController, shouldDiscard location: CLLocation)  -> Bool {
```

Comment:
Determining whether to show the weak GPS banner was not the only purpose of this method. We should still implement this method and delegate it to the receiver.

Reply:
👍 good call